src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingCTestConfiguration.kt

```diff
@@ -41,6 +41,9 @@ class ModelCheckingCTestConfiguration(testClass: Class<*>, iterations: Int, thre
     checkObstructionFreedom, hangingDetectionThreshold, invocationsPerIteration, guarantees, requireStateEquivalenceCheck,
     minimizeFailedScenario, sequentialSpecification, timeoutMs, eliminateLocalObjects, verboseTrace, customScenarios, recoverabilityModel) {
     override fun createStrategy(testClass: Class<*>, scenario: ExecutionScenario, validationFunctions: List<Method>,
-                                stateRepresentationMethod: Method?, verifier: Verifier): Strategy
-        = ModelCheckingStrategy(this, testClass, scenario, validationFunctions, stateRepresentationMethod, verifier, recoverabilityModel)
+                                stateRepresentationMethod: Method?, verifier: Verifier): Strategy = if (recoverabilityModel.crashes) {
```

Comment:
Probably, this code is better readable when `if` is on the next line and there are no brackets.
Usually you write `val map = if (sorted) TreeMap(...) else HashMap(..)` and not `if (sorted) { TreeMap(...) } else { HashMap(...) }`